examples/counter/Clarinet.toml

```diff
@@ -1,6 +1,20 @@
 [project]
 name = "counter"
+authors = []
+description = ""
+telemetry = false
+cache_dir = "./.requirements"
+boot_contracts = ["pox", "costs-v2", "bns"]
+
+# The counter project is being used by the CI, and having to rely on network
+# leads to flaky tests.
```

Comment:
Is this the right place for this comment. I think it's confusing here, especially since new developers may be looking here for examples.

Reply:
Addressed with https://github.com/hirosystems/clarinet/pull/333/commits/c03549d1fa3804faa9fbd19f01e0933d0051ae0f